code-challenges.test.js

```diff
@@ -23,32 +25,100 @@ const numbersArray2 = [24, 27, 30, 33, 36]
 
 // --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
-// a) Create a test with expect statements for each of the variables provided.
+// Psuedo: Create a function that takes in the object number 
+// Input: value of the object with string interpolation to % 3 === 0(modulator) & % 3 !==(Bang) to determine if the value is divisible by 3 or not.
+// Output: whether or not the objects value is divisible by 3 
 
-const object1 = { number: 15 }
-// Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
-// Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
-// Expected output: "-7 is not divisible by three"
+// a) Create a test with expect statements for each of the variables provided.
+        const object1 = { number: 15 }
+        // Expected output: "15 is divisible by three"
+        const object2 = { number: 0 }
+        // Expected output: "0 is divisible by three"
+        const object3 = { number: -7 }
+        // Expected output: "-7 is not divisible by three"
+
+
+describe("divBy3", () => {
+    it("returns if a number is divisible by 3 or not", () => {
+        // expect(divBy3(object1)).toEqual("Something is off")
+        expect(divBy3(object1)).toEqual("15 is divisible by three")
+        expect(divBy3(object2)).toEqual("0 is divisible by three")
+        expect(divBy3(object3)).toEqual("-7 is not divisible by three")
+    })
+})
 
 // b) Create the function that makes the test pass.
+const divBy3 = (divide) => {
+    console.log(divide)
+    if (divide.number % 3 === 0){
+        return `${divide.number} is divisible by three`
+    } else if (divide.number % 3 === 0){
+        return `${divide.numer} is divisble by three`
+    } else if (divide.number % 3 !== 0){
+        return `${divide.number} is not divisible by three`
+    } else {
+        return "Math not mathing"
+    }
+}
```

Comment:
Great job on this function! You don't need to check if the number % 3 === 0 twice, I would get rid of the else if and return on line 55-56. Other than that, looks good!